src/main/webapp/3Dscript.js

```diff
@@ -176,6 +172,15 @@ function plotOrientation() {
   orientation.rotation.y = THREE.Math.degToRad(poseTransform.rotate);
 }
 
+/* Global variables that will hold point informsation to be shown in GUI. */
+let time;
+let lat;
+let long;
+let alt;
+let roll;
+let yaw;
+let pitch;
```

Comment:
What kind of objects are these? Do they have to be global?

Reply:
in another PR i plan to follow david's suggestion and put them in an array bc i use them to update the gui.

Reply:
In the meantime can you comment with what kind of objects they are?

Reply:
plus i use in more than 1 function